### ISSUE TITLE
pgkit/client seems to depend on pgsql-ast-parser but does not list as dependency

### ISSUE THREAD

**Issue by bjoernwuest** [mmkal/pgkit#431]
Version: 0.2.4

Code:
```import {Client, createClient, sql} from "@pgkit/client";```

Output (cut):
```
23:07:51 [vite] Error when evaluating SSR module /src/sql/service.ts: failed to import "@pgkit/client"
|- Error: Cannot find module 'pgsql-ast-parser'
Require stack:
- C:\...\node_modules\.pnpm\@pgkit+client@0.2.4-4\node_modules\@pgkit\client\dist\naming.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Module._load (node:internal/modules/cjs/loader:1051:27)
    at Module.require (node:internal/modules/cjs/loader:1311:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\...\node_modules\.pnpm\@pgkit+client@0.2.4-4\node_modules\@pgkit\client\dist\naming.js:28:28)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)
    at Module.load (node:internal/modules/cjs/loader:1288:32)
    at Module._load (node:internal/modules/cjs/loader:1104:12)
    at Module.require (node:internal/modules/cjs/loader:1311:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\...\node_modules\.pnpm\@pgkit+client@0.2.4-4\node_modules\@pgkit\client\dist\index.js:20:14)
```

Quickfix: add dependency to own package.json.

**Comment by mmkal** [comment]
Thank you! I plan to remove this dependency, but for now you can try your workaround, or use `0.2.4-5` which I just published with it moved from `devDependencies` to `dependencies`

**Comment by mmkal** [comment]
Fixed in 0.2.4
